### PR TITLE
Feature/164 manage pie chart

### PR DESCRIPTION
**Pull request by DamienSAVOYAT**
# What does this PR accomplish?
Allow creation of PieChart with customizable options for DataLabelModel : 
- ShowDataLabel : Indicate if labels data will be showed
- ShowCatName : Indicate if category name will be showed
- ShowPercent : Indicate if percentage will be showed
- LabelPosition : Indicate position of label
- Separator :  Indicate separator of differents values (label, data, percent...)

Refactor code of BarChart

### CODE REVIEW

File: OpenXMLSDK.Engine/Drawing/Pictures/Model/PictureModel.cs [hunk 1]
```diff
@@ -1,13 +0,0 @@
-﻿using OpenXMLSDK.Engine.Packaging;
```
Comment (mathieumack) [inline OpenXMLSDK.Engine/Drawing/Pictures/Model/PictureModel.cs]: File deleted ?

File: OpenXMLSDK.UnitTest/ReportEngine/ReportEngineTest.cs [hunk 2]
```diff
@@ -259,7 +315,7 @@ private static Document GetTemplateDocument()
             // Template 1 :
 
             var paragraph = new Paragraph();
-            paragraph.ChildElements.Add(new Label() { Text = "Label wihtou special character (éèàù).", FontSize = "30", FontName = "Arial" });
+            paragraph.ChildElements.Add(new Label() { Text = "Label wihtout special character (éèàù).", FontSize = "30", FontName = "Arial" });
```
Comment (mathieumack) [inline OpenXMLSDK.UnitTest/ReportEngine/ReportEngineTest.cs]: without ?

File: OpenXMLSDK.Engine/Drawing/Pictures/Model/PictureModel.cs (hunk 1 above)
Comment (DamienSAVOYAT) [reply OpenXMLSDK.Engine/Drawing/Pictures/Model/PictureModel.cs]: Yes, useless.

File: OpenXMLSDK.UnitTest/ReportEngine/ReportEngineTest.cs (hunk 2 above)
Comment (DamienSAVOYAT) [reply OpenXMLSDK.UnitTest/ReportEngine/ReportEngineTest.cs]: Fix.

**Pull request merged**